Userland/Applications/KeyboardSettings/KeyboardSettingsWidget.cpp

```diff
@@ -308,6 +308,9 @@ void KeyboardSettingsWidget::set_keymaps(Vector<DeprecatedString> const& keymaps
 
 void KeyboardSettingsWidget::write_caps_lock_to_ctrl_sys_variable(bool caps_lock_to_ctrl)
 {
+    if (getuid() != 0)
+        return;
+
```

Comment:
This should be squashed into the first commit, but isn't needed if we use Escalator